public/login.css

```diff
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: radial-gradient(#9b30ff, #9225a7, #7f2092);
```

Comment:
All our colors should be in variables. read about [css variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).